flang/test/Lower/Intrinsics/atan2d.f90

```diff
@@ -0,0 +1,28 @@
+! RUN: bbc -emit-fir -hlfir=false %s -o - | FileCheck %s --check-prefixes="CHECK,CHECK-FAST"
+! RUN: %flang_fc1 -emit-fir -flang-deprecated-no-hlfir %s -o - | FileCheck %s --check-prefixes="CHECK,CHECK-FAST"
+
+
+function test_real4_all_args(y,x)
```

Comment:
Did you mean the name to be `test_real4` instead?

Reply:
Well spot! Yes, they should not have `_all_args`, same changes will be made on `atan2pi` as well.